### PR TITLE
 Naming update for Stacks rebrand #841 

### PR DESCRIPTION
**Pull request by friedger**
## Description

This PR replaces the use of the name `Blockstack` with `Stacks` where it makes sense.

Some naming changes require an additional noun like `Blockstack` in the context of authentication becomes `Stacks Auth`.

For details refer to issue #841 

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [x] API reference/documentation update
- [ ] Other

## Does this introduce a breaking change?
No

## Are documentation updates required?
No

## Testing information
Nothing special

## Checklist
- [x] Code is commented where needed
- [x] Unit test coverage for new or modified code paths
- [x] `npm run test` passes
- [x] Changelog is updated
- [x] Tag 1 of @agraebe @GinaAbrams

### CODE REVIEW

File: src/pages/data-storage/collections.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ particular. The following topics are covered:
 
 ## Understand how collections work
 
-One of Blockstack's goals is to give users true data ownership by enabling _data portability_. Data portability allows
+One of Stacks's goals is to give users true data ownership by enabling _data portability_. Data portability allows
```
Comment (GinaAbrams) [inline src/pages/data-storage/collections.md:21]: One of the goals of the Stacks ecosystem is to give

File: src/pages/data-storage/collections.md [hunk 2]
```diff
@@ -43,15 +43,15 @@ pre-release provides the `Contact` collection. A contact schema produces this st
 
 A collection schema is neither validated or enforced. The goal is to incentivize collection use rather that enforce use.
 
-Because malicious apps or apps with poor security controls may damage user data, Blockstack believes collections should
-include the ability for users to roll-back changes. For this reason, Blockstack supports an event log and rollback
+Because malicious apps or apps with poor security controls may damage user data, Stacks believes collections should
```
Comment (GinaAbrams) [inline src/pages/data-storage/collections.md:46]: This seems funky, can we remove mentions of "Stacks believes"

File: src/pages/data-storage/overview.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
+The following diagram depicts the Stacks architecture and Gaia's place in it:
 
-![Blockstack Architecture](/images/architecture.png)
+![Stacks Architecture](/images/architecture.png)
 
 Blockchains require consensus among large numbers of people, so they can be slow. Additionally, a blockchain is not designed to hold a lot of data. This means using a blockchain for every bit of data a user might write and store is expensive. For example, imagine if an application were storing every tweet in the chain.
 
-Blockstack addresses blockchain performance problems using a layered approach. The base layer consists of the Stacks blockchain and the Blockstack Naming System (BNS). The blockchain governs ownership of identities in the Blockstack network. Identities can be names such as domain names, usernames, or application names.
+Stacks addresses blockchain performance problems using a layered approach. The base layer consists of the Stacks blockchain and the Blockstack Naming System (BNS). The blockchain governs ownership of identities in the Stacks network. Identities can be names such as domain names, usernames, or application names.
```
Comment (GinaAbrams) [inline src/pages/data-storage/overview.md:26]: The Stacks blockchain addresses

File: src/pages/data-storage/overview.md [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
 
 Gaia stores data as a simple key-value store. When an identity is created, a corresponding data store is associated with that identity on Gaia. When a user logs into a dApp,
 the authentication process gives the application the URL of a Gaia hub, which
 then writes to storage on behalf of that user.
 
-Within Blockstack, then, the Stacks blockchain stores only identity data. Data created by the actions of an identity is stored in a Gaia Storage System. Each user has profile data. When a user interacts with a decentralized dApp that application stores application data on behalf of the user. Because Gaia stores user and application data off the blockchain, a Blockstack DApp is typically more performant than DApps created on other blockchains.
+Within Stacks, then, the Stacks blockchain stores only identity data. Data created by the actions of an identity is stored in a Gaia Storage System. Each user has profile data. When a user interacts with a decentralized dApp that application stores application data on behalf of the user. Because Gaia stores user and application data off the blockchain, a Stacks DApp is typically more performant than DApps created on other blockchains.
```
Comment (GinaAbrams) [inline src/pages/data-storage/overview.md:36]: The Stacks blockchain stores only

File: src/pages/authentication/overview.md [hunk 5]
```diff
@@ -120,18 +120,18 @@ The app private key is securely shared with the app on each authentication, encr
 
 Both the `authRequest` and the `authResponse` tokens are [JSON Web Tokens](https://jwt.io/), and they are passed via URL query strings.
 
-Blockstack's authentication tokens are based on the [RFC 7519 OAuth JSON Web Token (JWT)](https://tools.ietf.org/html/rfc7519)
+Stacks's authentication tokens are based on the [RFC 7519 OAuth JSON Web Token (JWT)](https://tools.ietf.org/html/rfc7519)
```
Comment (GinaAbrams) [inline src/pages/authentication/overview.md:123]: Stacks authentication tokens?

**Comment by GinaAbrams** [comment]
Added a couple small comments, looks good otherwise! 🙏

**Comment by friedger** [comment]
@GinaAbrams there is a paragraph about Blockstack PBC and the history. What should be done with it?

Also who issued the stacks tokens?

**Comment by friedger** [comment]
@GinaAbrams  I have addressed your comments.

What about "The Blockstack App" and the "Blockstack Browser"?

The the two section in the overview:

* Blockstack PBC section: https://github.com/blockstack/docs/blob/05ac3f2882773346cfc1a9f371132219d3ce1810/src/pages/ecosystem/overview.md#L16
* Blockstack Token LLC: https://github.com/blockstack/docs/blob/05ac3f2882773346cfc1a9f371132219d3ce1810/src/pages/ecosystem/overview.md#L38

**Comment by GinaAbrams** [comment]
Thanks Friedger! I think we can probably wait on making changes to the PBC overview or Token LLC overview until the Hiro change has been more finalized.

**Review by agraebe — APPROVED** [review]

**Comment by agraebe** [comment]
@GinaAbrams could you help with a final review? I think we could merge this today if all looks good

File: src/pages/data-storage/collections.md [hunk 6]
```diff
@@ -18,13 +18,13 @@ particular. The following topics are covered:
...
 
-One of Blockstack's goals is to give users true data ownership by enabling _data portability_. Data portability allows
+One of the goals of the Stacks ecosystem is to give users true data ownership by enabling _data portability_. Data portability allows
 users to login with their digital ID on any app and have access to the same data. For example, if a user adds a photo of
 a Hawaiian vacation in one app, that photo enters the user's data pool. Then, when the user opens a second app, that
 same photo is available to the second app because the user data, including the photo, is shared via the user's
 decentralized ID.
 
-How do collections work? Blockstack builds a library containing commonly used data schemes. Developers use these classes
+How do collections work? Hiro PBC builds a library containing commonly used data schemes. Developers use these classes
```
Comment (GinaAbrams) [inline src/pages/data-storage/collections.md:27]: Can we use Stacks here to remain consistent on the language 🙏

**Review by GinaAbrams — COMMENTED** [review]
Added a single small comment. Otherwise lgtm!

File: src/pages/data-storage/collections.md (hunk 6 above)
Comment (friedger) [reply src/pages/data-storage/collections.md]: Updated!

**Comment by agraebe** [comment]
fantastic, thanks for this @friedger. I'll merge once the conflicts are resolved. can you take a look?

**Comment by friedger** [comment]
done

**Comment by agraebe** [comment]
@aulneau could you help here as well? I can't merge this one for some reason

**Comment by agraebe** [comment]
@friedger I can't merge this PR because of a conflict. could you resolve/rebase so we can merge this? thanks!

**Comment by friedger** [comment]
@agraebe done

**Comment by CharlieC3** [comment]
@friedger I think I see what's going on WRT the merge conflict. There's been some commits to `master` since you created this PR, in order to get a smooth merge and fix this, you need to perform a manual _rebase_ in your feature branch. Unfortunately it seems the `master` branch was merged into this feature branch quite a few times. There's a way to fix this properly, but honestly with the amount of merges from `master` performed it'll be simpler to just save the changes you made in this PR and re-create the feature branch from the latest commit in `master`.

For future reference, when you want to pull updates from the `master` branch into your feature branch, use:
`git pull --rebase origin master`

This will lift up the stack of changes you've made in the feature branch, and tuck new master updates underneath them to maintain chronological order, then it places your commits back on top of the commit stack.

You'll then need to _force_ push your branch back up to your fork
`git push <my-fork-name> -f`

I do this everytime I pull in updates from a base branch, and it should help keep PRs clean/mergeable.

**Comment by friedger** [comment]
This PR is replaced by #900

**Pull request closed without merging**